[Rules] New templates for Transform Rules and Snippets (#18481)
cloudflare/cloudflare-docs: src/content/docs/rules/snippets/examples/hex-timestamp.mdx

---
type: example

> summary: Add a custom header into outgoing requests with the current timestamp in hexadecimal format.

Reviewer: ```suggestion
summary: Add a custom header to requests sent to the origin server with the current timestamp in hexadecimal format.
```

goal:
  - Manage headers
operation:
  - Request modification
products:
  - Snippets
pcx_content_type: example
title: Add HEX timestamp to a request header

> description: Add a custom header into outgoing requests with the current timestamp in hexadecimal format for debugging, tracking, or custom routing purposes.

Reviewer: ```suggestion
description: Add a custom header to requests sent to the origin server with the current timestamp in hexadecimal format for debugging, tracking, or custom routing purposes.
```
